src/unity/toolkits/recsys/recsys_model_base.cpp

```diff
@@ -1239,6 +1238,26 @@ std::shared_ptr<unity_sframe_base> recsys_model_base::recommend_extension_wrappe
 
 }
 
+std::shared_ptr<unity_sframe_base> recsys_model_base::get_num_users_per_item_extension_wrapper() const {
+
```

Comment:
Some documentation here would be helpful for people who inherit this code. One-line function headers for all the functions you added in this file (and even elsewhere) would be great. Furthermore, wherever you think something complicated and non-trivial is happening, feel free to add additional documentation. Think of how you can make the ramp-up time for the person who works on this code a year from now, easier!

Reply:
There isn't a documentation for this wrapper code because this is a hack adapted from @znation 's recommend_extension_wrapper() above to get sframe across over to python for specific methods. We don't want to encourage workarounds for each individual method and want to fix this issue in a separate PR. Thanks for bringing this up, I've added a comment above the code explaining what the wrapper is here for.